Shell/CmdFullWrite.cpp

```diff
@@ -0,0 +1,52 @@
+#pragma once
+#include "CmdFullWrite.h"
+
+#include <iostream>
+#include <string>
+#include <vector>
+
+#include "SSDInterface.h"
+
+CommandFullWrite::CommandFullWrite(SSDInterface* ssdInterface)
+    : ssd(ssdInterface) {}
+
+bool CommandFullWrite::Call(std::vector<std::string> program) {
+  if (program.size() != 2) {
+    printInvalidCommandMessage();
+    return false;
+  }
+
+  const std::string& value = program[VALUE_INDEX];
+  if (IsInvalidValue(value)) {
+    printInvalidCommandMessage();
+    return false;
+  }
+
+  for (int lba = 0; lba < 100; ++lba) {
```

Comment:
오프라인에서 논의한 대로, lba값은 unsigned로 통일하면 좋을 것 같습니다.

```suggestion
  for (unsigned int lba = 0; lba < 100; ++lba) {
```